pkg/webhook/admission/pod/storage_initializer_injector_test.go

```diff
@@ -519,6 +519,117 @@ func TestCustomSpecStorageUriInjection(t *testing.T) {
 	}
 }
 
+func TestCustomSpecStorageMountPathInjection(t *testing.T) {
+	scenarios := map[string]struct {
+		original                       *v1.Pod
+		expectedStorageMountPathEnvVar *v1.EnvVar
+		expectedMountPath              string
+	}{
+		"CustomSpecStorageMountPathSet": {
+			original: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						constants.StorageInitializerSourceUriInternalAnnotationKey: "pvc://mypvcname/some/path/on/pvc",
+					},
+				},
+				Spec: v1.PodSpec{
+					Containers: []v1.Container{
+						{
+							Name: constants.InferenceServiceContainerName,
+							Env: []v1.EnvVar{
+								{
+									Name:  constants.CustomSpecStorageMountPathKey,
+									Value: "/custom/models",
+								},
+							},
+						},
+					},
+				},
+			},
+			expectedStorageMountPathEnvVar: &v1.EnvVar{
+				Name:  constants.CustomSpecStorageMountPathKey,
+				Value: "/custom/models",
+			},
+			expectedMountPath: "/custom/models",
+		},
+		// "CustomSpecStorageMountPathEmpty": {
```

Comment:
do we still need this test?

Reply:
> do we still need this test?

My idea is this test is necessary as it verifies whether the user-defined storage mount path is correctly injected.